core/mondoo-tls-security.mql.yaml

```diff
@@ -55,83 +56,105 @@ policies:
           mondoo-tls-security-mitigate-beast: null
       - title: Valid TLS/SSL certificate
         asset_filter:
-          query: platform.family.contains('network')
+          query: asset.family.contains('network')
         scoring_queries:
           mondoo-tls-security-cert-domain-name-match: null
           mondoo-tls-security-cert-is-valid: null
           mondoo-tls-security-cert-no-cert-expired: null
+          mondoo-tls-security-cert-no-certs-expired: null
           mondoo-tls-security-cert-not-self-signed: null
           mondoo-tls-security-cert-not-revoked: null
           mondoo-tls-security-cert-no-weak-signature: null
 
 queries:
   - uid: mondoo-tls-security-cert-domain-name-match
     title: The certificate's domain name must match
+    impact: 50
     query: |
       tls.certificates.first.subject.commonName == platform.fqdn
   - uid: mondoo-tls-security-cert-is-valid
     title: The certificate is valid
+    impact: 95
     query: |
       tls.certificates.first {
+        subject.commonName
         notBefore < time.now
         notAfter - notBefore < 398*time.day
       }
   - uid: mondoo-tls-security-cert-no-cert-expired
-    title: None of the certificates have expired
+    title: Certificate has not expired
```

Comment:
Doe this one need an `impact`?

Reply:
It does not require impact, since the impact is defined by the query.